sycl/include/sycl/ext/intel/esimd/detail/memory_intrin.hpp

```diff
@@ -458,8 +456,7 @@ __esimd_scatter_scaled(__ESIMD_DNS::simd_mask_storage_t<N> pred,
     uint32_t width;
     std::mutex *mutexLock;
 
-    I->sycl_get_cm_buffer_params_index_ptr(surf_ind, &writeBase, &width,
-                                           &mutexLock);
+    I->sycl_get_cm_buffer_params_ptr(surf_ind, &writeBase, &width, &mutexLock);
```

Comment:
Why passing by pointer and not reference?

Reply:
Pointer is more clear & explicit on the caller side.

Reply:
Plus sycl_get_cm_buffer_params_ptr is a part of C interface, not C++.

Reply:
If it's C, then ok.